apps/crl.c

```diff
@@ -207,6 +207,8 @@ int crl_main(int argc, char **argv)
     if (argc != 0)
         goto opthelp;
 
+    if (digestname != NULL && !opt_md(digestname, &digest))
```

Comment:
not sure why you did this as an && and then do the other files differently?

Reply:
> not sure why you did this as an && and then do the other files differently?

Thanks for catching that.  I missed this.  Fixup commit coming.